esx_service/vmdk_ops.py

```diff
@@ -1318,10 +1325,11 @@ def disk_detach(vmdk_path, vm):
     if not device:
        # Could happen if the disk attached to a different VM - attach fails
        # and docker will insist to sending "unmount/detach" which also fails.
-       msg = "*** Detach failed: disk={0} not found. VM={1}".format(
-       vmdk_path, vm.config.uuid)
-       logging.warning(msg)
-       return err(msg)
+       # Or Plugin retrying operation due to socket errors #1076
```

Comment:
we actually do refer to github issues and PRs in different places, so while it may be good or bad, it is certainly consistent (there are 30 cases like that in 17 files in our repo, including ./vendor). BTW , what/where  is this coding convention asking to skip PR numbers ?